index.js

```diff
@@ -3,7 +3,7 @@ import express from 'express'
 const app = express()
 
 app.get('/', (req, res) => {
-  res.send('Hello World!')
+  res.send('Hello Wor ld!')
```

Comment:
[GPT-REVIEW] There seems to be a typo in the string 'Hello Wor ld!'. It should be 'Hello World!'. Please correct it.